### PR TITLE
added octave cheatsheet

### PR DESCRIPTION
**Pull request by Vaibhav-Dubey**
## What does this PR do?
This PR adds a cheat sheet for Octave language

### Why is this valuable (or not)?
this is valuable for math-cs graduates that work on octave for mathematical plotting and solving math problems
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: more/free-programming-cheatsheets.md [hunk 1]
```diff
@@ -284,6 +285,11 @@
 * [Quick Cheat Sheet for Mongo DB Shell commands](https://gist.github.com/michaeltreat/d3bdc989b54cff969df86484e091fd0c) - Michael Treat's Quick Cheat Sheet (GitHub Gist)
 
 
+### Octave
+
+* [Octave Cheat Sheet](https://www.lehman.edu/academics/cmacs/documents/refcard-a4.pdf) (PDF)
```
Comment (shrn01) [inline more/free-programming-cheatsheets.md:290]: LGTM

**Review by shrn01 — APPROVED** [review]

File: more/free-programming-cheatsheets.md (hunk 1 above)
Comment (eshellman) [inline more/free-programming-cheatsheets.md:290]:
```suggestion
* [Octave Cheat Sheet](https://www.lehman.edu/academics/cmacs/documents/refcard-a4.pdf) -  John
W. Eaton (PDF)
```
Please add author names where appropriate.

Comment (Vaibhav-Dubey) [reply more/free-programming-cheatsheets.md]: Oh thanks for this suggestion @eshellman , done the suggested changes :)

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**